ReplayKitExample/BroadcastExtension/SampleHandler.swift

```diff
@@ -0,0 +1,229 @@
+                                         nil,
+                                         &outPixelBuffer);
+        if (status != kCVReturnSuccess) {
+            print("Failed to create pixel buffer");
+        }
+
+        CVPixelBufferLockBaseAddress(outPixelBuffer!, []);
+
+        // Prepare source pointers.
+        var sourceImageY = vImage_Buffer(data: CVPixelBufferGetBaseAddressOfPlane(pixelBuffer, 0),
+                                         height: vImagePixelCount(CVPixelBufferGetHeightOfPlane(pixelBuffer, 0)),
+                                         width: vImagePixelCount(CVPixelBufferGetWidthOfPlane(pixelBuffer, 0)),
+                                         rowBytes: CVPixelBufferGetBytesPerRowOfPlane(pixelBuffer, 0))
+
+        var sourceImageUV = vImage_Buffer(data: CVPixelBufferGetBaseAddressOfPlane(pixelBuffer, 1),
+                                          height: vImagePixelCount(CVPixelBufferGetHeightOfPlane(pixelBuffer, 1)),
+                                          width:vImagePixelCount(CVPixelBufferGetWidthOfPlane(pixelBuffer, 1)),
+                                          rowBytes: CVPixelBufferGetBytesPerRowOfPlane(pixelBuffer, 1))
+
+        // Prepare out pointers.
+        var outImageY = vImage_Buffer(data: CVPixelBufferGetBaseAddressOfPlane(outPixelBuffer!, 0),
+                                      height: vImagePixelCount(CVPixelBufferGetHeightOfPlane(outPixelBuffer!, 0)),
+                                      width: vImagePixelCount(CVPixelBufferGetWidthOfPlane(outPixelBuffer!, 0)),
+                                      rowBytes: CVPixelBufferGetBytesPerRowOfPlane(outPixelBuffer!, 0))
+
+        var outImageUV = vImage_Buffer(data: CVPixelBufferGetBaseAddressOfPlane(outPixelBuffer!, 1),
+                                       height: vImagePixelCount(CVPixelBufferGetHeightOfPlane(outPixelBuffer!, 1)),
+                                       width:vImagePixelCount( CVPixelBufferGetWidthOfPlane(outPixelBuffer!, 1)),
+                                       rowBytes: CVPixelBufferGetBytesPerRowOfPlane(outPixelBuffer!, 1))
+
+
+        var error = vImageScale_Planar8(&sourceImageY, &outImageY, nil, vImage_Flags(0));
+        if (error != kvImageNoError) {
+            print("Failed to down scale luma plane ")
+            return;
+        }
+
+        error = vImageScale_CbCr8(&sourceImageUV, &outImageUV, nil, vImage_Flags(0));
+        if (error != kvImageNoError) {
+            print("Failed to down scale chroma plane")
+            return;
+        }
+
+        CVPixelBufferUnlockBaseAddress(outPixelBuffer!, []);
+        CVPixelBufferUnlockBaseAddress(pixelBuffer, []);
+
+        let time = CMSampleBufferGetPresentationTimeStamp(sampleBuffer);
+        let frame = TVIVideoFrame(timestamp: time,
+                                  buffer: outPixelBuffer!,
+                                  orientation: TVIVideoOrientation.up)
+
+        captureConsumer?.consumeCapturedFrame(frame!)
```

Comment:
Confirm the color space of the input frames, and at least add a comment if it is not sRGB / rec.709.